requirements/base.txt

```diff
@@ -11,6 +11,7 @@ whitenoise==3.2.2
 # Forms
 django-braces==1.10.0
 django-crispy-forms==1.6.1
+django_filter==1.0.4
```

Comment:
Why is this needed? Can't this be done simply using URL query params passed in the request and use Django object query?